### ISSUE TITLE
[MICROGRANT] Mint Condition

### ISSUE THREAD

**Issue by TheGodOfAwesome** [ipfs/devgrants#153]
### 1. What is your project? (max 100 words)

Mint Condition https://www.mintcondition.app/ 
Our product helps educators who want to monetize their by content and curriculum By adding new monetization options (e.g. after sale royalties) And increasing leaner incentive through collector resell. We are the front-end for Education Content hosted on IPFS.
We help creators mint their Education material (Courses, Tutorials, Interviews and Documentaries) as NFTs and allows collectors to learn, and then earn with the creators through a resale ecosystem built on the Blockchain. (Ethereum and Polygon)
We are a homegrown African solution to a global problem.

https://github.com/Mint-Condition
We are going to open source all the useful aspects that don't compromise the security of our users and their content but we are still deciding how to best do that.

https://www.mintcondition.app/

### 2. How are you planning to improve this project? (max 200 words)

For our beta we currently help our creators upload their long form content (we have already built tools for smaller file size projects and short form content using Pinata and Moralis ipfs tools) on a one on one basis but we are working on building tools to improve the long form content upload process  and to manage the content once created. (We see this as some the work we believe that will be of the most value open sourced)
 
### 3. Will the work be Open Source?

We are going to open source all the useful aspects that don't compromise the security of our users and their content but we are still deciding how to best do that. 

### 4. If selected, do you agree to complete weekly updates and a grant report upon conclusion?

Yes we would expect nothing less.
  
### 5. Does your proposal comply with our Community Code of Conduct?

No we do not violate the Code of conduct. We pay specific attention to ensure the ownership of the content on our site and that it does not violate ANY Copyright Laws. 

### 6. Links and submissions

  
* Have you submitted to the IPFS Community Showcase ([instructions](https://github.com/ipfs/community/blob/master/README.md#showcase-your-project))?
  

* Have you added your project to the [IPFS Ecosystem Directory](https://airtable.com/shrjwvk9pAeAk0Ci7)? 
Yes
  
* If your project began at a hackathon, have you submitted it for the relevant Protocol Labs prizes? Include links here if available:
Our co-founder and CTO has won a prize in an IPFS hackathon EthGlobal BuildQuest but we were working on this project prior to this.
(project: [showcase.ethglobal.com/buildquest/caste-m7bgy](https://showcase.ethglobal.com/buildquest/caste-m7bgy))
  
* Have you filled out the [Interplanetary Builder Feedback Survey](https://airtable.com/shrDZMizx03jOa4mQ)? 
Yes
  
### Additional questions:
* For each team member(s), please list name, email, Github account, and role in the project.
Kuzivakwashe Muvezwa https://github.com/TheGodOfAwesome muvezwakuzivakwashe@yahoo.com
Michael Musekiwa michaelmusekiwa@gmail.com

* If your project was created as part of an event or hackathon:
  * What was the name of the event? (e.g. ETHGlobal NFTHack, Cal Hacks hello:world, Chainlink, CivHacks, GameDevJ, ETHGlobal Scaling Ethereum)
  * Please link to your hackathon submission, proving you're a team that submitted for a Protocol Labs prize.

**Comment by ErinOCon** [comment]
Hi @TheGodOfAwesome, Thank you for your proposal. We will review this and get back to you, on this thread, with a status update or questions.

**Comment by ErinOCon** [comment]
@TheGodOfAwesome, thank you again for your submission! Can you provide further detail on how Filecoin and/or IPFS will be integrated into this work?

I would also like to note that projects funded through this program should be open-source prior to grant approval and payment. If you have any questions regarding this criteria, please do not hesitate to reach out!

**Comment by TheGodOfAwesome** [comment]
Hi @ErinOCon we help Creators offer their communities access to their Educational content e.g. Courses, Tutorials, Papers, Documentaries etc. using NFTs. All the afore mentioned content is hosted on IPFS and our website works as a marketplace that helps these creators monetize said content using NFT gating. 
So essentially if you were a youtuber who maybe gets 3000 total views on your Woodworking content which wouldn't create a sustainable living. Mint Condition would help you

1. Upload said content to IPFS.
2. Avail access to this content via an NFT.
3. Make it easy for your community to buy this NFT.
4. Allow owners of the NFT to then consume your content
5. And if the consumer has completed the video or course they can sell the NFT and allow another person to gain access to the content as well with Mint Condition managing the sale and the creator royalty payouts (royalties will eventually move on chain)

The benefit being that creators with niche audiences would have an easier time monetizing their content. 
In the example I gave if only 50 people of the 3000 people who normally watch your videos where willing to mint the NFT for the video that would be better monetization for the creator.
We also hope to create an incentive for leaners where once the learn from your content the can recoup part of their initial investment in the content by selling this NFT to this video and therefore sharing the creator's work at the same time allowing a creator to monetize again via resale royalties.

We intended to open source already but we were just unsure of how to go about it because we have one or two decisions to make around how we would open source without exposing our creator's  NFT gated content. But being aware of it being a requirement of this grant we are happy to move this step forward much faster and open source our platform prior to receiving the grant.

**Comment by ErinOCon** [comment]
@TheGodOfAwesome, thank you very much for the additional detail! Can the linked repo be made public? This will help us continue our review.

**Comment by ErinOCon** [comment]
Thanks, @TheGodOfAwesome! Once this is done, our team will proceed with the review.

**Comment by TheGodOfAwesome** [comment]
Hi @ErinOCon sorry for the delayed reply. We had to go back and figure out how we were going to open source the project and ultimately we realized that we couldn't open source it and maintain the integrity of our users content. So as a result we are withdrawing our application, sorry for the inconvenience and thank you so much for the your time and help. If its okay with you I am planning on opening a different application with my hackathon project which was an IPFS prize winner instead.
